requirements/static/ci/common.in

```diff
@@ -1,7 +1,24 @@
 # Requirements in this file apply to all platforms.
 # We can also exclude platforms from the requirements using markers, but if a requirement only applies
 # to a particular platform, please add it to the corresponding `<platform>.in` file in this directory.
-azure==4.0.0; sys_platform != 'win32'
+#azure==5.0.0; sys_platform != 'win32'
```

Comment:
Why is this one commented out and the rest of the direct dependencies added in this file?